src/MFPEncoder.ts

```diff
@@ -171,6 +170,7 @@ export default class MFPBaseEncoder {
     const layer = layerState.layer;
 
     if (layer instanceof VectorTileLayer) {
+      const {MVTEncoder} = await import('@geoblocks/print');
```

Comment:
Is it well-supported by different bundler?

Reply:
https://caniuse.com/es6-module-dynamic-import